data/json/items/gun/762x25.json

```diff
@@ -28,5 +28,30 @@
     "magazines": [
       [ "762x25", [ "ppshmag", "ppshdrum" ] ]
     ]
+  },
+  {
+    "id": "tokarev",
+    "type": "GUN",
+    "symbol": "(",
+    "color": "dark_gray",
+    "name": "Tokarev TT-30",
+    "name_plural": "Tokarev TT-30",
```

Comment:
It may be a good idea to rename the Tokarev to TT-33. Also, the Tokarev isn't the standard sidearm of the Chinese military anymore.